python/gtsam/tests/test_GaussianFactorGraph.py

```diff
@@ -88,5 +88,22 @@ def test_linearMarginalization(self):
         self.assertAlmostEqual(EXPECTEDM[1], m[1], delta=1e-8)
         self.assertAlmostEqual(EXPECTEDM[2], m[2], delta=1e-8)
 
+    def test_ordering(self):
```

Comment:
Could you add more comments about what is happening and what the bug is you uncovered with this? Also maybe add output so I can help think about the cause, or if you know what the problem is, let us know :-)

Reply:
The issue seems to be in the way the Debug configuration is handled in the wrapper. Release versions of this same test passes no problem but we get a segfault in the Debug version. Pretty confusing.

Reply:
Output isn't helpful, but here it is:

```
python/gtsam/tests/test_GaussianFactorGraph.py Fatal Python error: Segmentation fault

Current thread 0x00007fa529fd2280 (most recent call first):
  File "/home/varun/borglab/gtsam/build/python/gtsam/tests/test_GaussianFactorGraph.py", line 103 in test_ordering
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/unittest/case.py", line 633 in _callTestMethod
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/unittest/case.py", line 676 in run
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/unittest/case.py", line 736 in __call__
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/_pytest/unittest.py", line 321 in runtest
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/_pytest/runner.py", line 162 in pytest_runtest_call
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/pluggy/callers.py", line 187 in _multicall
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/pluggy/manager.py", line 84 in <lambda>
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/pluggy/manager.py", line 93 in _hookexec
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/pluggy/hooks.py", line 286 in __call__
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/_pytest/runner.py", line 255 in <lambda>
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/_pytest/runner.py", line 311 in from_call
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/_pytest/runner.py", line 254 in call_runtest_hook
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/_pytest/runner.py", line 215 in call_and_report
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/_pytest/runner.py", line 126 in runtestprotocol
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/_pytest/runner.py", line 109 in pytest_runtest_protocol
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/pluggy/callers.py", line 187 in _multicall
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/pluggy/manager.py", line 84 in <lambda>
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/pluggy/manager.py", line 93 in _hookexec
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/pluggy/hooks.py", line 286 in __call__
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/_pytest/main.py", line 348 in pytest_runtestloop
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/pluggy/callers.py", line 187 in _multicall
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/pluggy/manager.py", line 84 in <lambda>
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/pluggy/manager.py", line 93 in _hookexec
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/pluggy/hooks.py", line 286 in __call__
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/_pytest/main.py", line 323 in _main
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/_pytest/main.py", line 269 in wrap_session
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/_pytest/main.py", line 316 in pytest_cmdline_main
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/pluggy/callers.py", line 187 in _multicall
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/pluggy/manager.py", line 84 in <lambda>
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/pluggy/manager.py", line 93 in _hookexec
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/pluggy/hooks.py", line 286 in __call__
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/_pytest/config/__init__.py", line 162 in main
  File "/home/varun/.pyenv/versions/3.8.11/lib/python3.8/site-packages/_pytest/config/__init__.py", line 185 in console_main
  File "/home/varun/.pyenv/versions/3.8.11/bin/pytest", line 8 in <module>
[1]    3008430 segmentation fault (core dumped)  pytest  -s
```

Reply:
Commenting out this particular test section since it is time-consuming and out of scope for this PR.